tests/src/snapshots/fe_compiler_tests__compile_errors__issue_451.snap

```diff
@@ -0,0 +1,15 @@
+---
+source: tests/src/compile_errors.rs
+expression: "error_string(&path, &src)"
+
+---
+error: a function with the same name already exists
+  ┌─ fixtures/compile_errors/issue_451.fe:4:5
+  │  
+4 │ ╭     pub def bar:
+5 │ │      s
+  │ ╰──────^ Conflicting definition of contract `bar`
```

Comment:
I guess "contract" is a copy-paste issue neither of us noticed last time. Probably worth fixing now.

Reply:
Oops, right :+1: